classes/author.rb

```diff
@@ -0,0 +1,14 @@
+class Author
+  attr_reader :id, :first_name, :last_name, :items
```

Comment:
Please note that `first_name` and `last_name` should have both getters and setters because they are marked with `+` in the UML diagram.
This means you can leave `id` and `items` on line 2, but remove `first_name` and `last_name` to a separate line:
`attr_accessor :first_name, :last_name`

The whole code will look like:
```
attr_reader :id, :items
attr_accessor :first_name, :last_name
```